src/function/diff_outputs.rs

```diff
@@ -25,7 +25,12 @@ where
         revisions: &mut QueryRevisions,
     ) {
         // Iterate over the outputs of the `old_memo` and put them into a hashset
```

Comment:
I would find it useful if we could extend the comment so that it explains why we ignore the generation part.